docs/requirement-sets/common/nested-app-auth-requirement-sets.md

```diff
@@ -19,16 +19,15 @@ Office Add-ins run across multiple versions of Office. The following table lists
 > [!IMPORTANT]
 >
 > - Currently, the NestedAppAuth 1.1 requirement set is supported in Office on the web only for documents that are opened from Microsoft SharePoint Online and OneDrive.
-> - In Outlook, the NestedAppAuth 1.1 requirement set isn't supported if the add-in is loaded in an Outlook.com or Gmail mailbox.
 
 ## Supported accounts and hosts
 
-NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn't support Azure Active Directory B2C for business-to-consumer identity management scenarios. The following table explains the current support by platform. Platforms listed as generally available (GA) are ready for production usage in your add-in.
+NAA supports both Microsoft Accounts (MSA), such as personal outlook.com accounts, and Microsoft Entra ID (work/school) identities. It doesn't support Azure Active Directory B2C for business-to-consumer identity management scenarios. The following table explains the current support by platform. Platforms listed as generally available (GA) are ready for production usage in your add-in.
```

Comment:
```suggestion
NAA supports both Microsoft Accounts (MSA), such as personal outlook.com accounts, and Microsoft Entra ID (work/school) identities. It doesn't support Microsoft Entra B2C for business-to-consumer identity management scenarios. The following table explains the current support by platform. Platforms listed as generally available (GA) are ready for production usage in your add-in.
```
See https://learn.microsoft.com/en-us/entra/fundamentals/new-name#glossary-of-updated-terminology.

Reply:
The product is still using this name so probably should stick with Azure Active Directory B2C until it gets changed.